src/main/java/io/opentracing/contrib/jdbc/JdbcTracingUtils.java

```diff
@@ -38,7 +38,7 @@ static Span buildSpan(String operationName,
       boolean withActiveSpanOnly,
       Set<String> ignoredStatements,
       Tracer tracer) {
-    if (withActiveSpanOnly && tracer.activeSpan() == null) {
+    if (!TracingDriver.isTraceEnabled() || withActiveSpanOnly && tracer.activeSpan() == null) {
```

Comment:
@osvaldopina  could you change it to 
```
if (!TracingDriver.isTraceEnabled() || (withActiveSpanOnly && tracer.activeSpan() == null)) {
```
to be more clear

Reply:
much better indeed